### ISSUE TITLE
[Bug] - Build error when building with the partial-tracing feature

### ISSUE THREAD

**Issue by dlmanning** [aembke/fred.rs#222]
Redis version - N/A
Platform - macos
Using Docker and/or Kubernetes - no
Rust version 1.76

Trying to build a project using fred fails with the `partial-tracing` feature.

**To Reproduce**
1. create a new empty project
2. add `fred = { version = "8.0.3", features = ["partial-tracing"] }` to dependencies in `Cargo.toml`
3. attempt to build

**Logs**
```
   Compiling fred v8.0.3
error[E0308]: mismatched types
   --> /Users/david/.cargo/registry/src/index.crates.io-6f17d22bba15001f/fred-8.0.3/src/utils.rs:468:34
    |
468 |     args_span.record("num_args", command.args().len());
    |               ------             ^^^^^^^^^^^^^^^^^^^^ expected `&_`, found `usize`
    |               |
    |               arguments to this method are incorrect
    |
    = note: expected reference `&_`
                    found type `usize`
note: method defined here
   --> /Users/david/.cargo/registry/src/index.crates.io-6f17d22bba15001f/fred-8.0.3/src/trace/disabled.rs:20:10
    |
20  |   pub fn record<Q: ?Sized, V: ?Sized>(&self, _field: &Q, _value: &V) -> &Self {
    |          ^^^^^^                                          ----------
help: try removing the method call
    |
468 -     args_span.record("num_args", command.args().len());
468 +     args_span.record("num_args", command.args());
    |

For more information about this error, try `rustc --explain E0308`.
error: could not compile `fred` (lib) due to 1 previous error
```

**Additional context**
Have attempted to make sure I'm not missing something stupid here. Please let me know if I am.

**Comment by aembke** [comment]
Thanks @dlmanning , this should be fixed in 8.0.4.

**Comment by dlmanning** [comment]
Thanks for the quick turnaround 😄
